docs/api/products.rst

```diff
@@ -36,150 +171,385 @@ Response will contain a paginated list of products.
-                "href":"\/api\/products\/?page=12"
-            },
-            "next":{
-                "href":"\/api\/products\/?page=2"
+	"id": 52,
+	"code": "b32c7f49-0693-3f3a-a57b-e86b4eb00e96",
+	"attributes": [
+		{
+			"code": "t_shirt_material",
+			"name": "T-Shirt material",
+			"value": "Potato 100%",
+			"id": 111
+		}
+	],
+	"variants": [
+		{
+			"id": 196,
+			"on_hold": 0,
+			"tracked": false
+		},
+		{
+			"id": 197,
+			"on_hold": 0,
+			"tracked": false
+		},
+		{
+			"id": 198,
+			"on_hold": 0,
+			"tracked": false
+		}
+	],
+	"translations": {
+		"en_US": {
+			"locale": "en_US",
+			"id": 52,
+			"name": "T-Shirt \"voluptate\"",
+			"slug": "t-shirt-voluptate",
+			"description": "Libero nihil odit exercitationem repellendus consequuntur libero aut.",
+			"short_description": "Expedita laudantium ea quod molestias totam."
+		}
+	},
+	"_links": {
+		"self": {
+			"href": "/api/v1/products/52"
+		}
+	}
+}
+
+Creating Product
+----------------
+
+Definition
+..........
+
+.. code-block:: text
+
+    POST /api/v1/products/
+
++---------------+----------------+--------------------------------------------------------+
+| Parameter     | Parameter type | Description                                            |
++===============+================+========================================================+
+| Authorization | header         | Token received during authentication                   |
++---------------+----------------+--------------------------------------------------------+
+| name          | request        | Name of creating product                               |
++---------------+----------------+--------------------------------------------------------+
+| code          | request        | **(unique)** Product identifier                        |
++---------------+----------------+--------------------------------------------------------+
+| slug          | request        | **(unique)**  Name converted to valid format for URL   |
++---------------+----------------+--------------------------------------------------------+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/v1/products/
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng"
+        -H "Content-Type: application/json"
+        -X POST
+        --data '
+            {
+                "translations": {
+                    "en_US": {
+                        "name": "Truck Simulator",
+                        "slug": "truck-simulator"
+                    }
+                },
+                "code": "TS3"
+            }
+        '
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 201 CREATED
+
+.. code-block:: json
+
+    {
+        "id": 64,
+        "code": "TS3",
+        "attributes": [],
+        "variants": [],
+        "translations": {
+            "en_US": {
+                "locale": "en_US",
+                "id": 64,
```

Comment:
In api's tests we can have pretty view, but in documentation IMO we should show the true response, not the pretty one.